### DISCUSSION TITLE
[VOTE] Release Apollo Java version 2.2.0

### DISCUSSION THREAD

**Discussion by nobodyiam** [apolloconfig/apollo-java#47] [General]
Hi All,

@apolloconfig/pmc @apolloconfig/committers This is a call for vote to release Apollo Java version 2.2.0.

Release Note:

 * https://github.com/apolloconfig/apollo-java/releases/tag/v2.2.0

Maven 2 staging repository:

- https://oss.sonatype.org/service/local/repositories/comctrip-1081/content/com/ctrip/framework/apollo/

You may add the following to pom.xml to test the new artifacts.

```xml
<repositories>
    <repository>
        <id>sonatype-nexus-release</id>
        <url>https://oss.sonatype.org/service/local/repositories/comctrip-1081/content</url>
        <releases>
            <enabled>true</enabled>
        </releases>
        <snapshots>
            <enabled>true</enabled>
        </snapshots>
    </repository>
</repositories>
```

Release Tag :

 * https://github.com/apolloconfig/apollo-java/releases/tag/v2.2.0

Release CommitID :

 * https://github.com/apolloconfig/apollo-java/commit/2c2b833dc4c68bdcb3da841d4d1436347d946677

Guide to build the release from source:

 * mvn clean package -DskipTests=true

Voting will start now (2023/12/04) and will remain open for at least 72 hours, Request all PMC members and committers to give their vote.
[ ] +1 Release this package.
[ ] +0 No opinion.
[ ] -1 Do not release this package because....

**Comment by klboke** [comment]
+1 ,I checked the list of release features and there is no problem

**Comment by hezhangjian** [comment]
+1

**Comment by JaredTan95** [comment]
+1

**Comment by zouyx** [comment]
+1 approved

**Comment by Anilople** [comment]
+1 approved

**Comment by vdiskg** [comment]
+1 approved

**Comment by lepdou** [comment]
+1 approved

**Comment by nobodyiam** [comment]
The vote to release Apollo Java version 2.2.0 has passed with seven +1 and zero -1.
